akka-sample-cqrs-scala/src/main/scala/sample/cqrs/otherservice/Main.scala

```diff
@@ -0,0 +1,120 @@
+package sample.cqrs.otherservice
```

Comment:
For simplicity I placed the other consuming service in the same project. That would ofc be a completely separate projection if done for real.